packages/create-cloudflare/templates/hello-world-python/py/wrangler.toml

```diff
@@ -3,3 +3,97 @@ name = "<TBD>"
 main = "src/entry.py"
 compatibility_flags = ["python_workers"]
 compatibility_date = "<TBD>"
+
+# Variable bindings. These are arbitrary, plaintext strings (similar to environment variables)
+# Note: Use secrets to store sensitive data.
```

Comment:
should we also add a link to the secrets doc?

(https://developers.cloudflare.com/workers/configuration/secrets/)

Reply:
I like this suggestion.

Reply:
That is a good suggestion indeed.

I'll do this in a followup PR since I want to make the change to all the `wrangler.toml` files. In that same PR I'd like to update the worker templates to use `build-cf-types` as well and get rid of the redundant definition/docs in their source files.